referral-form-data-process/serverless.yml

```diff
@@ -5,6 +5,7 @@ provider:
   runtime: nodejs14.x
   region: eu-west-2
   lambdaHashingVersion: 20201221
+  timeout: 30
```

Comment:
Does it require such a high timeout?

Reply:
This was randomly selected while debugging. I can remove this and we can probably add a better timeout value when we've got the template and have a better idea of the lambda execution time.

Reply:
Done in ef2e296